mmdeploy/backend/base/backend_utils.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import importlib
+from abc import ABCMeta, abstractstaticmethod
+from typing import Any, Optional, Sequence
+
+
+class BaseBackendUtils(metaclass=ABCMeta):
+    """Abstract interface of backend utils."""
+
+    @abstractstaticmethod
+    def build_wrapper(backend_files: Sequence[str],
+                      device: str = 'cpu',
+                      input_names: Optional[Sequence[str]] = None,
+                      output_names: Optional[Sequence[str]] = None,
+                      deploy_cfg: Optional[Any] = None,
+                      **kwargs):
+        """Build the wrapper for the backend model.
+
+        Args:
+            backend_files (Sequence[str]): Backend files.
+            device (str, optional): The device info. Defaults to 'cpu'.
+            input_names (Optional[Sequence[str]], optional): input names.
+                Defaults to None.
+            output_names (Optional[Sequence[str]], optional): output names.
+                Defaults to None.
+            deploy_cfg (Optional[Any], optional): The deploy config. Defaults
+                to None.
+        """
+
+
+class BackendUtilsRegistry:
```

Comment:
could we add a unit test for this class?

Reply:
Not necessary, the feature would be used in other ut.